cranelift/codegen/src/isa/aarch64/inst.isle

```diff
@@ -1143,6 +1144,8 @@
     (Fmin)
     ;; Floating-point multiply
     (Fmul)
+    ;; Floating-point fused multiply-add vectors
+    (Fmla)
```

Comment:
Could we add a comment here (and also on `bsl` while we're at it) to note that the opcode must only be used with the `vec_rrr_inplace` constructor?

(Note below on a better approach here by separating the types for safety, but we don't need to do it right away)